js/about/preferences.js

```diff
@@ -260,9 +260,20 @@ class BitcoinDashboard extends ImmutableComponent {
   get ledgerData () {
     return this.props.ledgerData
   }
-  get overlayContent () {
+  get bitcoinOverlayContent () {
     return <iframe src={this.ledgerData.get('buyURL')} />
   }
+  get qrcodeOverlayContent () {
+    return <div>
+      <img src={this.ledgerData.get('paymentIMG')} title='Brave wallet QR code' />
+      <div className='bitcoinQRTitle' data-l10n-id='bitcoinQR' />
+    </div>
+  }
+  get qrcodeOverlayFooter () {
```

Comment:
i don't think the QR code overlay should show the coinbase logo since it doesn't depend on coinbase at all